### PR TITLE
Making Spanish CI leg as the defaut target CI

### PR DESCRIPTION
**Pull request by Anipik**
Merge After https://github.com/dotnet/corefx/pull/28952 and https://github.com/dotnet/corefx/pull/29009

### CODE REVIEW

**Comment by Anipik** [comment]
@dotnet-bot test Windows x64 Debug Build
@dotnet-bot test Windows x86 Release Build

File: buildpipeline/windows.groovy [hunk 1]
```diff
@@ -74,13 +74,11 @@ simpleNode('Windows_NT','latest') {
                 {
                     targetHelixQueues = ['Windows.10.Amd64.Open',
                                          'Windows.7.Amd64.Open',
-                                         'Windows.81.Amd64.Open',]
+                                         'Windows.81.Amd64.Open',
+                                         'Windows.10.Amd64.ClientRS3.ES.Open',]
```
Comment (safern) [inline buildpipeline/windows.groovy]: Do we want to get rid of the default Windows.10.Amd64.Open? It should be the same as the Windows.10 in spanish.

cc: @danmosemsft

Comment (Anipik) [reply buildpipeline/windows.groovy]: @danmosemsft  why its a good idea to get rid of English Language Build ?

**Comment by danmosemsft** [comment]
I suggest holding this until 2.2

**Comment by Anipik** [comment]
@dotnet-bot test Windows x64 Debug Build
@dotnet-bot test Windows x86 Release Build

**Comment by danmosemsft** [comment]
Odd.
```
OSX.1013.Amd64.Open-x64-Debug
Get Repro environment
Unhandled Exception of Type Xunit.Sdk.TrueException
Message :
Timed out after 60000ms waiting for remote process 61432
Expected: True
Actual:   False
Stack Trace :
   at System.Diagnostics.RemoteExecutorTestBase.RemoteInvokeHandle.Dispose(Boolean disposing) in /Users/dotnet-bot/j/workspace/dotnet_corefx/master/osx-TGroup_netcoreapp+CGroup_Debug+AGroup_x64+TestOuter_false_prtest/src/CoreFx.Private.TestUtilities/src/System/Diagnostics/RemoteExecutorTestBase.cs:line 241
   at System.Diagnostics.RemoteExecutorTestBase.RemoteInvokeHandle.Dispose() in /Users/dotnet-bot/j/workspace/dotnet_corefx/master/osx-TGroup_netcoreapp+CGroup_Debug+AGroup_x64+TestOuter_false_prtest/src/CoreFx.Private.TestUtilities/src/System/Diagnostics/RemoteExecutorTestBase.cs:line 228
   at System.Tests.EnvironmentTests.CurrentDirectory_SetToValidOtherDirectory() in /Users/dotnet-bot/j/workspace/dotnet_corefx/master/osx-TGroup_netcoreapp+CGroup_Debug+AGroup_x64+TestOuter_false_prtest/src/System.Runtime.Extensions/tests/System/EnvironmentTests.cs:line 41
```

**Comment by danmosemsft** [comment]
@Anipik if you like, but it's not actionable except for potentially looping the test locally on your macbook. You can ignore the test for this change.

**Comment by Anipik** [comment]
@danmosemsft can you approve these changes ?

**Comment by Anipik** [comment]
@dotnet-bot test OSX x64 Debug Build

**Pull request merged**